packages/core/src/input/pointer/PointerManager.ts

```diff
@@ -244,7 +244,7 @@ export class PointerManager {
           // TODO: Only check which colliders have listened to the input.
           if (this._engine.physicsManager.raycast(camera.viewportPointToRay(point, ray), hitResult)) {
             return hitResult.entity;
-          } else if (camera.clearFlags === CameraClearFlags.DepthColor) {
+          } else if (camera.clearFlags === CameraClearFlags.Color) {
```

Comment:
Maybe should `camera.clearFlags & CameraClearFlags.Color`?

Reply:
Done.